config/config.go

```diff
@@ -485,8 +485,10 @@ type Performance struct {
 	CrossJoin             bool    `toml:"cross-join" json:"cross-join"`
 	RunAutoAnalyze        bool    `toml:"run-auto-analyze" json:"run-auto-analyze"`
 	DistinctAggPushDown   bool    `toml:"distinct-agg-push-down" json:"distinct-agg-push-down"`
-	CommitterConcurrency  int     `toml:"committer-concurrency" json:"committer-concurrency"`
-	MaxTxnTTL             uint64  `toml:"max-txn-ttl" json:"max-txn-ttl"`
+	// Whether enable projection push down for coprocessors (both tikv & tiflash), default false.
+	ProjectionPushDown   bool   `toml:"projection-push-down" json:"projection-push-down"`
```

Comment:
Would we remove this session var after 1~2 sprints once it's stable?

Reply:
Some statistics' construction would rely on it. So i want it to always be on.

Reply:
yes, that's possible, we'll add more extensive tests relying on projection pushdown soon, and it will strengthen the robustness of the change, and I can foresee the default enablement after that.